src/collection/collection.service.ts

```diff
@@ -1249,6 +1255,48 @@ export class CollectionService {
 		}
 	}
 
+	static async fetchBookmarkedCollectionsByOwner(
+		user: Avo.User.User,
+		offset: number,
+		limit: number | null,
+		order: any,
+		filterString: string | undefined
+	): Promise<Avo.Collection.Collection[]> {
+		let variables: any;
+		try {
+			const trimmedFilterString = filterString && filterString.trim();
+			const filterArray: any[] = [];
+			if (trimmedFilterString) {
+				filterArray.push({
+					bookmarkedCollection: { title: { _ilike: `%${trimmedFilterString}%` } },
+				});
+			}
+			variables = {
+				offset,
+				limit,
+				order,
+				owner_profile_id: getProfileId(user),
+				filter: filterArray.length ? filterArray : {},
+			};
+			const response = await dataService.query({
+				variables,
+				query: GET_BOOKMARKED_COLLECTIONS_BY_OWNER,
+			});
+
+			if (response.errors) {
+				throw new CustomError('graphql response contains errors', null, { response });
+			}
+
+			const bookmarks = get(response, 'data.app_collection_bookmarks', []);
```

Comment:
Avoid lodash get so we get better type checking once we add strong database types
```suggestion
			const bookmarks = response?.data?.app_collection_bookmarks || [];
```